src/Cake.Core/Constants.cs

```diff
@@ -11,7 +11,7 @@ internal static class Constants
         public const ConsoleColor DefaultConsoleColor = (ConsoleColor)(-1);
 
         public static readonly Version LatestBreakingChange = new Version(0, 26, 0);
```

Comment:
Not related, but is this used anymore?

Reply:
Yes `LatestBreakingChange` will throw an error `LatestPotentialBreakingChange` will log an warning.

Reply:
Shouldn't `LatestBreakingChange` be something like 4.0.0 then?

Reply:
No, most add-ins compiled for older versions of Cake will still work just fine. Cake NuGet will pick the highest available TFM. If we set 4.0.0, they won't even load, causing support hassle for us for something that might potentially break, hence the warning and recommendation were added a few years ago. We've removed nothing known to break from core interfaces since 0.26.0, but we've added stuff that could potentially break if someone provided their own implementation of core features.

Reply:
So couldn't we remove it all together then?